app/src/main/java/com/vultisig/wallet/presenter/vault_setting/vault_edit/VaultRenameScreen.kt

```diff
@@ -1,204 +1,39 @@
-import androidx.compose.foundation.layout.fillMaxSize
-import androidx.compose.foundation.layout.fillMaxWidth
-import androidx.compose.foundation.layout.imePadding
-import androidx.compose.foundation.layout.padding
-import androidx.compose.foundation.layout.wrapContentHeight
-import androidx.compose.foundation.shape.RoundedCornerShape
-import androidx.compose.foundation.text2.BasicTextField2
-import androidx.compose.material.icons.Icons
-import androidx.compose.material.icons.automirrored.filled.KeyboardArrowLeft
-import androidx.compose.material3.Card
-import androidx.compose.material3.CardDefaults
-import androidx.compose.material3.CenterAlignedTopAppBar
-import androidx.compose.material3.ExperimentalMaterial3Api
-import androidx.compose.material3.Icon
-import androidx.compose.material3.IconButton
-import androidx.compose.material3.MaterialTheme
-import androidx.compose.material3.Scaffold
-import androidx.compose.material3.SnackbarHost
-import androidx.compose.material3.SnackbarHostState
-import androidx.compose.material3.Text
-import androidx.compose.material3.TopAppBarDefaults
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.LaunchedEffect
 import androidx.compose.runtime.collectAsState
 import androidx.compose.runtime.getValue
-import androidx.compose.runtime.remember
-import androidx.compose.ui.Alignment
-import androidx.compose.ui.Modifier
-import androidx.compose.ui.graphics.Color
-import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.res.stringResource
-import androidx.compose.ui.text.font.FontWeight
-import androidx.compose.ui.tooling.preview.Preview
-import androidx.compose.ui.unit.dp
 import androidx.hilt.navigation.compose.hiltViewModel
 import androidx.navigation.NavHostController
-import androidx.navigation.compose.rememberNavController
 import com.vultisig.wallet.R
-import com.vultisig.wallet.common.asString
 import com.vultisig.wallet.presenter.vault_setting.vault_edit.VaultEditEvent.OnNameChange
 import com.vultisig.wallet.presenter.vault_setting.vault_edit.VaultEditEvent.OnSave
-import com.vultisig.wallet.presenter.vault_setting.vault_edit.VaultEditUiEvent.ShowSnackBar
-import com.vultisig.wallet.ui.components.MultiColorButton
-import com.vultisig.wallet.ui.theme.Theme
-import com.vultisig.wallet.ui.theme.dimens
+import com.vultisig.wallet.ui.components.NamingComponent
 
 @Composable
 internal fun VaultRenameScreen(
     navController: NavHostController,
     viewModel: VaultRenameViewModel = hiltViewModel()
 ) {
     val uiModel by viewModel.uiModel.collectAsState()
-    val snackBarHostState = remember { SnackbarHostState() }
-    val context = LocalContext.current
 
     LaunchedEffect(key1 = Unit) {
         viewModel.loadData()
-        viewModel.channelFlow.collect { event ->
-            when (event) {
-                is ShowSnackBar ->
-                    snackBarHostState.showSnackbar(event.message.asString(context))
-            }
-        }
```

Comment:
Did that part of the code get lost or become irrelevant?

Reply:
Fixed

Reply:
Check again please